Description: don't enable onboard in GNOME sessions since caribou is being
 used and it leads to have two onscreen keyboards loaded at the same time
Bug: https://launchpad.net/bugs/1721125
--- a/data/onboard-autostart.desktop.in
+++ b/data/onboard-autostart.desktop.in
@@ -9,4 +9,4 @@
 X-Ubuntu-Gettext-Domain=onboard
 AutostartCondition=GSettings org.gnome.desktop.a11y.applications screen-keyboard-enabled
 X-GNOME-AutoRestart=true
-OnlyShowIn=GNOME;Unity;MATE;
+OnlyShowIn=Unity;MATE;
